.travis.yml

```diff
@@ -1,15 +1,14 @@
 language: node_js
 node_js:
-  - '12'
+  - stable
 env:
-  - EXTENSION_ID=lmjdlojahmbbcodnpecnjnmlddbkjhnn
+  - EXTENSION_ID='lmjdlojahmbbcodnpecnjnmlddbkjhnn'
```

Comment:
This isn't necessary, right?

Reply:
Unless there are any spaces, it's not. A matter of personal preference, if I can say.

Not seriously attached to it though, open to drop it if it is that big of an issue. 🤷‍♂️